grails-app/conf/Config.groovy

```diff
@@ -167,6 +167,7 @@ environments {
         gogoduck.url = "http://${localhostAddress}:8300/go-go-duck"
         geonetwork.url = "https://catalogue-imos.aodn.org.au/geonetwork"
         geonetwork.version = 2
+        geonetwork.searchService = "xml.search.summary"
         gogoduck.filenamePrepend = "IMOS_aggregation"
 
         if (geonetwork.version == 3) {
```

Comment:
I'd just set the search service according to the version and not require anyt cloud deply config until we work out its necessary.

Reply:
Note this would have to go at the end of the file after reading cloud-deploy config